### PR TITLE
user-crud: Connect vue and express app.

### PR DESCRIPTION
**Pull request by nikhilatcompro**

### CODE REVIEW

File: vuejs/user-display-app/client/src/components/MatListItem.vue [hunk 1]
```diff
@@ -43,7 +43,8 @@ import MatMenu from './MatMenu.vue';
 
 <style scoped>
   .list-item {
-    position: relative;
     overflow: inherit;
+    padding: 6px 0 6px 0;
```
Comment (chakshuujawa) [inline vuejs/user-display-app/client/src/components/MatListItem.vue:47]: use rem

File: vuejs/user-display-app/client/src/components/ShowUser.vue [hunk 2]
```diff
@@ -66,8 +70,10 @@
         this.$router.push('/users');
       }
     },
-    created () {
-      this.user = this.$store.state.users[this.$route.params.id - 1];
+    async created () {
+      console.log(this.$route.params.id);
+      this.user = await UserService.getUser(this.$route.params.id);
+      console.log(this.user);
```
Comment (chakshuujawa) [inline vuejs/user-display-app/client/src/components/ShowUser.vue:76]: Is this console needed? Pls check

File: vuejs/user-display-app/client/src/components/MatListItem.vue (hunk 1 above)
Comment (nikhilatcompro) [reply vuejs/user-display-app/client/src/components/MatListItem.vue]: done

File: vuejs/user-display-app/client/src/components/ShowUser.vue [hunk 3]
```diff
@@ -66,8 +70,10 @@
         this.$router.push('/users');
       }
     },
-    created () {
-      this.user = this.$store.state.users[this.$route.params.id - 1];
+    async created () {
+      console.log(this.$route.params.id);
```
[the remark this answers is not in the document]
Comment (nikhilatcompro) [reply vuejs/user-display-app/client/src/components/ShowUser.vue]: removed

File: vuejs/user-display-app/client/src/components/ShowUser.vue (hunk 2 above)
Comment (nikhilatcompro) [reply vuejs/user-display-app/client/src/components/ShowUser.vue]: not needed. removed.

**Pull request merged**